test/Microsoft.Framework.FileSystemGlobbing.Tests/TestUtility/RunWhenWhenDirectoryInfoWorksAttribute.cs

```diff
@@ -21,7 +22,7 @@ public string SkipReason
         {
             get
             {
-                return "CoreCLR DirectoryInfo incorrectly text directory existence.";
+                return "Issue: \"DirectoryInfo.Exists returns incorrect results when it is cast from FileSystemInfo.\" is fixed. Please remove the workaround in the produce code and this test case.";
```

Comment:
Why not let the test fail when the coreclr fix is available at our end so that we automatically know. With skipping a test we need to watch if the test is skipped.